pulsar-client-cpp/lib/Commands.h

```diff
@@ -49,7 +49,7 @@ class Commands {
     };
     enum WireFormatConstant
     {
-        MaxMessageSize = (5 * 1024 * 1024 - (10 * 1024)),
+        DefaultMaxMessageSize = (5 * 1024 * 1024 - (10 * 1024)),
         MaxFrameSize = (5 * 1024 * 1024)
```

Comment:
We should making sure these are aligned with the constants in Java. Eg. there we have 5Mb max msg size with an additional padding of 10K on top of it for the frame size.